Cargo.toml

```diff
@@ -38,6 +38,7 @@ chrono = "0.3"
 env_logger = "0.3"
 futures-cpupool = "0.1"
 clap = "2.0"
+service-fn = { git = "https://github.com/tokio-rs/service-fn" }
```

Comment:
I prefer we pin to a specific hash if there's no crate. Is that OK or do you still want to track master?